types/expo/index.d.ts

```diff
@@ -1394,66 +1395,136 @@ export namespace SQLite {
  */
 export interface SvgCommonProps {
     fill?: string;
-    fillOpacity?: number;
+    fillOpacity?: number | string;
     stroke?: string;
-    strokeWidth?: number;
-    strokeOpacity?: number;
+    strokeWidth?: number | string;
+    strokeOpacity?: number | string;
     strokeLinecap?: string;
     strokeLineJoin?: string;
     strokeDasharray?: any[];
     strokeDashoffset?: any;
-    x?: Axis;
```

Comment:
Is `Axis` used anywhere now? if not can you remove it altogether?

Reply:
It is used elsewhere:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/expo/index.d.ts#L35
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/expo/index.d.ts#L951
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/expo/index.d.ts#L1173

In fact, `Axis` parts could just be replaced with `number` since it is just that. But it's a bit out of scope of this PR, I can submit a separate PR to address it.

Reply:
Yep, `Axis` type uses for all coordinates properties. @martynaskadisa was saying true in above comment, this is just number. I think this is the logical alias.

Reply:
`expo` uses `react-native-svg` library.
https://github.com/react-native-community/react-native-svg/blob/master/lib/props.js#L4
As we see `numberProp` is `number | string` type, I agree with @martynaskadisa.

I introduced `Axis` type just for all coordinates properties. Maybe it is not a good idea use it with components.

Reply:
@KonstantinKai Do you think this PR is good to go?

Reply:
@martynaskadisa, yeah, why not. All who uses `svg` components this PR will help.